vite.config.js

```diff
@@ -14,6 +14,11 @@ export default defineConfig(({ command, mode }) => {
   // Load the environment variables, whether or not they are prefixed with REACT_APP_
   const env = loadEnv(mode, process.cwd(), ['REACT_APP_', '']);
 
+  // Work around for process.env not being loaded correctly in knex library
+  const clientEnv = Object.fromEntries(
+    Object.entries(env).map(([key, value]) => [`process.env.${key}`, JSON.stringify(value)]),
```

Comment:
Why do we use a fully qualified `process.env.${key}` rather than just `key` and store it under the `process.env` key within define by before? I am sure this is right, just want to understand more.

Reply:
This is because in knex, they have a piece of logic that assign something to process.env like screenshot below (which is so hacky). If we only define `process.env`, it also converted the whole `process.env` in knex library into the actual process.env that we define in vite, and throw a syntax error 🤦 
<img width="549" alt="Screenshot 2025-06-02 at 11 06 54 am" src="https://github.com/user-attachments/assets/c1fa7840-ef74-4c0e-a2ef-15a1101cac7e" />

The only way to work around that is I had to avoid defining the whole `process.env`, and instead define the value for every `process.env.${key}` like below. I don't really like it, but that's the only way to make it work 🤦